src/coreclr/src/vm/exceptionhandling.cpp

```diff
@@ -1417,6 +1422,12 @@ void ExceptionTracker::InitializeCrawlFrame(CrawlFrame* pcfThisFrame, Thread* pT
     INDEBUG(memset(pcfThisFrame, 0xCC, sizeof(*pcfThisFrame)));
     pcfThisFrame->pRD = pRD;
 
+    // Clear various flags because the above INDEBUG sets them to true
```

Comment:
I believe that we would need to clear these even if the above INDEBUG did not exist. There is nothing initializing them.

Reply:
I'll remove "because the above INDEBUG sets them to true" from the comments.